app/models/manageiq/providers/amazon/storage_manager/s3/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,8 @@
+module ManageIQ::Providers::Amazon::CloudManager::CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def raw_delete
+    raise "CloudObjectStoreContainer has no #{ui_lookup(:table => "ext_management_systems")}, unable to delete it" unless ext_management_system
+    with_provider_object(&:delete!)
+  end
```

Comment:
@miha-plesko - is there a status property that needs to be updated to reflect a deletion is in progress until the next refresh happens?